### DISCUSSION TITLE
fee estimate endpoint

### DISCUSSION THREAD

**Discussion by Talej** [ACINQ/eclair#2472] [Q&A]
Is there a REST endpoint that can be used to get an estimated fee either given node id and amount or invoice?

**Comment by DerEwige** [comment]
I don't think you can't get it directly, but you can get a route with the fee information for each step, which you have to add up, to get total fee.

https://acinq.github.io/eclair/#findroute (nodeId + invoice)
https://acinq.github.io/eclair/#findroutetonode  (nodeid + amount in msat)

With format = "full" you get base fee and percentage fee for each hope and can add them up.
Hope that helps you

**Comment by Talej** [comment]
thanks for that. I was missing the format = "full" option to find those details
